app/views/admin/attachments/confirm_destroy.html.erb

```diff
@@ -2,6 +2,11 @@
 <% content_for :page_title, "Delete attachment" %>
 <% content_for :title, "Delete attachment" %>
 <% content_for :title_margin_bottom, 6 %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: attachable_attachments_path(attachable)
+  } %>
+<% end %>
```

Comment:
Do we def want a backlink and cancel link on the same page?

Reply:
Oh, good catch, sorry, have removed it again